sources/api/storewolf/build.rs

```diff
@@ -1,11 +1,32 @@
-// Automatically generate README.md from rustdoc.
-
+/// This build script generates README.md from rustdoc, like our other crates, but also generates
+/// a unified TOML file representing the default settings for the system.  The contents of that
+/// file are used by storewolf to populate the defaults into the data store on a new system.
+///
+/// The goal of generating the defaults file here is to allow variants to break up and share
+/// groups of default settings, without having to ship those files in the OS image.  Specifically,
+/// we read any number of files from a defaults.d directory in the variant's model directory and
+/// merge later entries into earlier entries, so later files take precedence.
+use merge_toml::merge_values;
+use snafu::ResultExt;
 use std::env;
-use std::fs::File;
+use std::fs::{self, File};
 use std::io::Write;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
+use toml::{map::Map, Value};
+use walkdir::WalkDir;
+
+/// A variant stores its default settings in .toml files in this directory.  It can link to shared
+/// files if desired.  Entries are sorted by filename, and later entries take precedence.
```

Comment:
I would love to have these few sentences more visibly documented so a user understands how the files are merged and how they should set up the filenames in the variant's directory.

It probably makes sense in the README in the models directory.